examples/example_config.yaml

```diff
@@ -68,28 +68,18 @@ slurm:
 ###### BEGIN PARAMETER CONFIGURATION ######
 
 fixed:
-  reg_scale: 0.0
-  keep_prob: 0.5
   max_epochs: 500
   patience: 10
```

Comment:
Should we remove some hyperparameters that don't show anything new? Reducing this to the essentials should make the example easier to understand.

Reply:
Fixed in a40eca8.